android/build.gradle

```diff
@@ -55,6 +55,7 @@ android {
         disable "GradleDependency"
         disable "NewerVersionAvailable"
         disable "DuplicatePlatformClasses"      // xpp3 added by azure-identity
```

Comment:
Added the suppressions to the `lint.xml` to prevent annotations all over our code. Also added a baseline for known failures like the InvalidPackage to a baseline file. We can build on it as we get more errors.